docs/index.rst

```diff
@@ -3,24 +3,97 @@ Welcome to Dockstore Documentation!
 
 .. note:: Our code lives on GitHub at `dockstore/dockstore <https://github.com/dockstore/dockstore>`_ and `dockstore/dockstore-ui2 <https://github.com/dockstore/dockstore-ui2>`_.
 
-Dockstore is an open platform used by the GA4GH for sharing Docker-based tools
-described with either the Common Workflow Language (CWL), the Workflow Description Language (WDL),
-or Nextflow (NFL).
+Dockstore is an open platform used by the `GA4GH <https://www.ga4gh.org/>`__ for sharing Docker-based tools, workflows, and services.
+For tools and workflows, we support Common Workflow Language (CWL), Workflow Description Language (WDL), Nextflow (NFL), and Galaxy.
```

Comment:
I think the `the`s that were there before are correct.

We support the Common Workflow Language, or, We support CWL.

Reply:
The fact we have CWL in parenthesis after that would imply "the CWL" to my brain, but that could just be me, so I checked their docs to see what they do. Turns out it seems the don't use "the CWL" nor "the Common Workflow Language".

https://www.commonwl.org/v1.0/CommandLineTool.html has two instances of there being no article...
1. "Common Workflow Language" (title)
2. "Michael R. Crusoe <email>, Common Workflow Language project"

...and four instances where there is an article, but it seems the article applies to another part of the sentence.
3. "This specification defines the Common Workflow Language (CWL) Command Line Tool Description" -- article could apply to "Command Line Tool Description"
4. "This document is the product of the Common Workflow Language working group." -- article applies to "working group"
5. "The Common Workflow Language (CWL) working group is an informal..." -- article applies to "working group"
6. "The Common Workflow Language Command Line Tool Description is designed to..." -- article could apply to "Command Line Tool Description"

What's your take on this?